docs/src/man/split_apply_combine.md

```diff
@@ -3,24 +3,43 @@
 Many data analysis tasks involve splitting a data set into groups, applying some functions to each of the groups and then combining the results. A standardized framework for handling this sort of computation is described in the paper "[The Split-Apply-Combine Strategy for Data Analysis](http://www.jstatsoft.org/v40/i01)", written by Hadley Wickham.
 
 The DataFrames package supports the split-apply-combine strategy through the `by` function, which is a shorthand for `groupby` followed by `map` and/or `combine`. `by` takes in three arguments: (1) a `DataFrame`, (2) one or more columns to split the `DataFrame` on, and (3) a specification of one or more functions to apply to each subset of the `DataFrame`. This specification can be of the following forms:
-1. a `col => function` pair indicating that `function` should be called with the vector of values for column `col`, which can be a column name or index
-2. a `cols => function` pair indicating that `function` should be called with a named tuple holding columns `cols`, which can be a tuple or vector of names or indices
-3. several such pairs, either as positional arguments or as keyword arguments (mixing is not allowed), producing each a single separate column; keyword argument names are used as column names
-4. equivalently, a (named) tuple or vector of such pairs
-5. a function which will be called with a `SubDataFrame` corresponding to each group; this form should be avoided due to its poor performance
-
-In all of these cases, the function can return either a single row or multiple rows, with a single or multiple columns:
+1. a `col => function` pair indicating that `function` should be called with the vector
+   of values for column `col`, which can be a column name or index
+2. a `cols => function` pair indicating that `function` should be called with
+   positional arguments holding columns `cols`, which can be a any valid column selector
```

Comment:
Maybe merge these two cases? The distinction was needed before as a named tuple was passed for 2, but now 1 is a special case of 2.

Reply:
OK - done.